### PR TITLE
FID-31 feat(Footer): Modification of footer

### PR DESCRIPTION
**Pull request by MatheusFrate**
# 🎬 Filmaro 🎬 - Pull Request

## Description:

The Footer was modified according to the mockup presented in Figma
## Type of change:

- [x] New feature (This PR adds a new functionality to the project)
- [ ] Technical debt (This PR improve features already created)
- [ ] Bug fix (This PR covers solving a bug)
- [ ] Documentation (This PR require a documentation update, i.e. the class doc)
- [ ] Issue resolution (This PR solve an github open issue)
[Link to solved Issue](#IssueLinkGoesHere)

## Checklist:

- [x] I created a responsive component.
- [x] My changes do not generate a new warning or issue.
- [x] I deleted all the code lines that are not being used.

👩🏽‍💻~ Filmaro Team ~👨🏽‍💻

### CODE REVIEW

**Review by vaaaarlos — APPROVED** [review]
Aprovando, mas lembre-se de adicionar blank line no final do arquivo da próxima vez.

**Review by JoaoWudarski — APPROVED** [review]

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -6,12 +6,29 @@ export const Footer = () =>{
...
-        <span>{footer()}</span>
+        <div className='social'> 
+          <b style={{color: '#6A0DAD'}}>Filmaro</b> <br/>
+          <span>the next generation of movie catalog and information</span> 
+        </div>
+        <div className='options'>
+          <div className='optionsItens' ><b style={{ color: '#6A0DAD'}}> Pages </b>
+            <br/><a href='/home'> Home</a>
+            <br/><a href='/series'> Series</a>
+            <br/><a href='/movies'> filmes</a>
```
Comment (Im0tec) [inline src/components/Footer/Footer.js:22]: Filmes ta com "F" minúsculo. F

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -6,12 +6,29 @@ export const Footer = () =>{
...
+            <br/><a href='/streamings/Popular'> Em alta</a>
+          </div>
+          <div className='optionsItens' ><b style={{ color: '#6A0DAD'}}> Profile </b>
+            <br/><span> Login</span>
```
Comment (Im0tec) [inline src/components/Footer/Footer.js:26]: Descobri uma maneira melhor de implementar isso aqui, mas nunca testei. Se der vê se funfa isso aqui:
```js
{ 
   ['Login', 'Register', 'Add List'].map((item) => (
      <br><span key={item}>{item}</span> 
   ))
}

**Review by Im0tec — APPROVED** [review]
Nicee
download

File: src/components/Footer/Footer.js (hunk 2 above)
Comment (MatheusFrate) [reply src/components/Footer/Footer.js]: coloquei dessa forma, tanto na coluna de pages quanto na de profile

**Review by vaaaarlos — APPROVED** [review]

**Review by Im0tec — APPROVED** [review]
arranhou minhas costas quando eu passei com minha tropa













ain pretuuu

**Review by Im0tec — APPROVED** [review]

File: src/components/TitlePage/WebTitlePage/WebTitlePage.js [hunk 3]
```diff
@@ -65,17 +65,16 @@ export const WebTitlePage = ({buildVideoURL, titleInformation, getBrTitle, IMAGE
       }
       setScrollx(x);
     }
-    
+    console.log(showVideo && buildVideoURL())
```
Comment (MatheusFrate) [inline src/components/TitlePage/WebTitlePage/WebTitlePage.js:68]: @Im0tec ?

**Review by vaaaarlos — APPROVED** [review]

**Review by JoaoWudarski — APPROVED** [review]
ó O BONITÃOOOO 8==D

**Pull request merged**